src/sql/base/browser/ui/taskbar/actionbar.ts

```diff
@@ -128,15 +142,84 @@ export class ActionBar extends ActionRunner implements IActionRunner {
 		this._actionsList = document.createElement('ul');
 		this._actionsList.className = 'actions-container';
 		this._actionsList.setAttribute('role', 'toolbar');
+		this._actionsList.id = 'actions-container';
 		if (this._options.ariaLabel) {
 			this._actionsList.setAttribute('aria-label', this._options.ariaLabel);
 		}
 
 		this._domNode.appendChild(this._actionsList);
 
+		if (this._collapseOverflow) {
+			this._overflow = document.createElement('ul');
+			this._overflow.id = 'overflow';
+			this._overflow.className = 'overflow';
+			this._domNode.appendChild(this._overflow);
+		} else {
+			this._actionsList.style.flexWrap = 'wrap';
+		}
+
 		container.appendChild(this._domNode);
 	}
 
+	@debounce(300)
+	private resizeToolbar() {
+		let width = document.getElementById('actions-container').offsetWidth;
+		let fullWidth = document.getElementById('actions-container').scrollWidth;
+
+		// collapse actions that are beyond the width of the toolbar
+		if (width < fullWidth) {
+			// create '•••' more element if it doesn't exist yet
+			if (!this._moreItemElement) {
+				this._moreItemElement = document.createElement('li');
+				this._moreItemElement.className = 'action-item more';
+				this._moreItemElement.setAttribute('role', 'presentation');
+				this._moreItemElement.id = 'more';
+				const innerText: HTMLElement = document.createElement('a');
+				innerText.setAttribute('role', 'button');
+				innerText.innerHTML = '•••';
```

Comment:
are these emoji? please use the vscode's secondary commands's '...' string

Reply:
Ok changed it. Previously it was what we used in schema compare and dacpac extension for the open file buttons. 
![image](https://user-images.githubusercontent.com/31145923/78064789-1c20b800-7347-11ea-930e-439d8c74b3d3.png)